src/main/kotlin/racingcar/car/Car.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.car
+
+class Car(
+    private val engine: Engine,
+    private var _distance: Int = 0
+) {
```

Comment:
```suggestion
class Car(
    private val engine: Engine,
    distance: Int = 0
) {
    private var _distance = distance
```
변수명에 언더바(_)를 붙이는 방법은 backing property 기법을 사용할 때 사용합니다.